mayastor/src/rebuild.rs

```diff
@@ -5,6 +5,8 @@ use snafu::{ResultExt, Snafu};
 use spdk_sys::spdk_get_thread;
 use std::{cell::UnsafeCell, collections::HashMap, fmt};
 
+use futures::{channel::mpsc, StreamExt};
```

Comment:
Please consider using the crossbeam channels for this they are considered to be far more efficient at various aspects.

Reply:
I actually started with the crossbeam which we currently use for something already but it didn't seem to be using futures so ended up looking up async-std and finally this future:channel:mpsc. I'll have another look at the crossbeam ones.